style.css

```diff
@@ -161,6 +155,18 @@ body {
   flex-grow: 0;
 }
 
+.work {
```

Comment:
- [ ] You should horizontally center the works section as in the Figma design.
#### Design
![image](https://user-images.githubusercontent.com/44057863/170737932-4ed7ccf8-262c-4f2e-b1ec-04de2df3c2c1.png)

#### Impl
![image](https://user-images.githubusercontent.com/44057863/170737817-981bfa66-c4d8-4d63-83c2-ca0980a79b16.png)

Reply:
done